03_how_ln_works.asciidoc

```diff
@@ -89,518 +93,526 @@ Also, in case you are not familiar with P2PKH addresses and the basic format and
-* Your channel partner might have breached the protocol either due to a software bug or on purpose forcing you to protect your funds.
+* You want to reduce the balance held on your Lightning channels for security reasons and want to send funds to "cold storage"
+* Your channel partner becomes unresponsive for a long time and you cannot use the channel anymore.
+* The channel is not being used often because your channel partner is not a well-connected node, so you want to use the funds for another channel with a better-connected node.
+* Your channel partner has breached the protocol either due to a software bug or on purpose forcing you to close the channel to protect your funds.
 
 There are 3 ways to close a payment channel:
 
-* the good way - mutual close
-* the bad way - force close
-* the ugly way - protocol breach
-
-Different conditions require one or the other way to close a channel.
-For example, if your channel partner is offline you will not be able to follow "the good way" because a mutual close cannot be done without a cooperating partner.
-The good news is that your Lightning Network software will most likely automatically select the best closing mechanism available to you under your specific conditions. 
-
-Whether you as a user implicitly close a channel or the Lightning Network software automatically closes a channel for you due to a detected abnormality (e.g. cheating partner), it is usually the Lightning Network software that choses the appropriate way to close.
-
-===== Examining the good way - mutual close
-The preferred and good way to close a channel is the mutual close.
-When you decide that you want to close a channel, your Lightning Network node will inform your channel partner about your intention.
-Now the channel will be prepared for shutting down.
-No new routing attempts will be accepted from either channel partner and the ongoing routing attempts will be settled or removed after they timed out.
-Once no further routing attempts are pending, the closing transaction is prepared.
-This transaction is similar to the commitment transaction.
-It has the same balance as the last commitment transaction but no outputs are encumbered with a time lock.
-As finalizing the routing attempts can take some time, a mutual close can also take some time.
-The on-chain transaction fees of the on-chain transaction for closing the channel in a mutual way are being paid by the channel partner who opened the channel and not by the person who initiated the closing procedure.
-As both channel partners sign the on-chain transaction they have the chance to pay small fees for the Bitcoin transaction by using their on-chain fee estimator.
-Even though there is a potential waiting time, this type of channel close is usually faster than closing the bad way.
-
-===== Examining the bad way - force close
-In case your node cannot engage in a "mutual close" you will have to do a "force close".
```

Comment:
maybe move the removed text to another chapter where we need more detail instead of just deleting it. if it needs rewrite at least we have a reminder?